tests/e2e/taqueria-plugin-flextesa.spec.ts

```diff
@@ -116,102 +117,82 @@ describe('E2E Testing for taqueria flextesa plugin sandbox starts/stops', () =>
+		const dockerContainerStopTest = await getContainerName(sandboxName);
+		expect(dockerContainerStopTest).toBe('');
 	});
 
 	test('Verify that taqueria flextesa plugin will return "The local sandbox was not running." if user tries to call stop on sandbox that is not running', async () => {
-		try {
-			// 1. Run stop sandbox local on sandbox that is not running and verify result
-			const sandboxWasNotRunning = await exec('taq stop sandbox local', { cwd: `./${taqueriaProjectPath}` });
-			expect(sandboxWasNotRunning.stdout).toEqual('The local sandbox was not running.\n');
-		} catch (error) {
-			throw new Error(`error: ${error}`);
-		}
+		// 1. Run stop sandbox local on sandbox that is not running and verify result
+		const sandboxWasNotRunning = await exec('taq stop sandbox local', { cwd: `./${taqueriaProjectPath}` });
+		expect(sandboxWasNotRunning.stdout).toEqual('The local sandbox was not running.\n');
 	});
 
 	test('Verify that taqueria flextesa plugin will return "The local sandbox is not running." if user tries to retrieve list of accounts that is not running', async () => {
-		try {
-			// 1. Run list accounts command on sandbox that is not running and verify result
-			const stdoutSandboxIsNotRunning = await exec('taq list accounts local', { cwd: `./${taqueriaProjectPath}` });
-			expect(stdoutSandboxIsNotRunning.stderr).toEqual('The local sandbox is not running.\n');
-		} catch (error) {
-			throw new Error(`error: ${error}`);
-		}
+		// 1. Run list accounts command on sandbox that is not running and verify result
+		const stdoutSandboxIsNotRunning = await exec('taq list accounts local', { cwd: `./${taqueriaProjectPath}` });
+		expect(stdoutSandboxIsNotRunning.stderr).toEqual('The local sandbox is not running.\n');
 	});
 
 	test('Verify that taqueria flextesa plugin will return "Already running." if sandbox has started" if user tries to call start sandbox twice', async () => {
-		try {
-			sandboxName = 'local';
+		sandboxName = 'local';
 
-			// 1. Run sandbox start command
-			await exec(`taq start sandbox ${sandboxName}`, { cwd: `./${taqueriaProjectPath}` });
+		// 1. Run sandbox start command
+		await exec(`taq start sandbox ${sandboxName}`, { cwd: `./${taqueriaProjectPath}` });
 
-			// 2.  Run start command second time and verify the output
-			const sandboxStart = await exec(`taq start sandbox ${sandboxName}`, { cwd: `./${taqueriaProjectPath}` });
-			expect(sandboxStart.stdout).toEqual('Already running.\n');
-		} catch (error) {
-			throw new Error(`error: ${error}`);
-		}
+		// 2.  Run start command second time and verify the output
+		const sandboxStart = await exec(`taq start sandbox ${sandboxName}`, { cwd: `./${taqueriaProjectPath}` });
+		expect(sandboxStart.stdout).toEqual('Already running.\n');
 	});
 
-	test('Verify that Taqueria accepts any origin and does not emit any CORS related errors', async () => {
+	test.skip('Verify that Taqueria accepts any origin and does not emit any CORS related errors', async () => {
```

Comment:
Do we still want to skip this test?

Reply:
Good catch. I'll revert that and see what happens.